client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -152,6 +152,16 @@ public abstract void mapperEnd(
       String applicationId, int shuffleId, int mapId, int attemptId, int numMappers)
       throws IOException;
 
+  // Report partition locations written by the completed Map Partition type mapper
+  public abstract void mapPartitionMapperEnd(
```

Comment:
sorry I am a little confused with the name, would you please describe more details about the method?

Reply:
Just modified the method description to make it clearer